cirq-google/cirq_google/engine/stream_manager.py

```diff
@@ -121,6 +121,14 @@ def __init__(self, grpc_client: quantum.QuantumEngineServiceAsyncClient):
         # interface.
         self._response_demux = ResponseDemux()
         self._next_available_message_id = 0
+        # Construct queue in AsyncioExecutor to ensure it binds to the correct event loop, since it
+        # is used by asyncio coroutines.
+        self._request_queue: asyncio.Queue[
+            Optional[quantum.QuantumRunStreamRequest]
+        ] = self._executor.submit(self._make_request_queue).result()
```

Comment:
nit: I think the type will be inferred here, so you could omit the type annotation:
```suggestion
        self._request_queue = self._executor.submit(self._make_request_queue).result()
```